packages/grpc-web/exports.js

```diff
@@ -6,12 +6,16 @@
  */
 goog.module('grpc.web.Exports');
 
+const CallOptions = goog.require('grpc.web.CallOptions');
+const MethodDescriptor = goog.require('grpc.web.MethodDescriptor');
 const GrpcWebClientBase = goog.require('grpc.web.GrpcWebClientBase');
+const RpcError = goog.require('grpc.web.RpcError');
 const StatusCode = goog.require('grpc.web.StatusCode');
-const MethodDescriptor = goog.require('grpc.web.MethodDescriptor');
 const MethodType = goog.require('grpc.web.MethodType');
```

Comment:
nit: do you mind help sort this line as well? :)

Reply:
The order reflects exports from `index.d.ts`? Is this ok?

Reply:
Ah thanks for the considerations :)

Personally i still prefer the exports to be alphabetically ordered in this file, given the connection to `index.d.ts` is an obscure one and would be harder to maintain (especially if we don't even document it here).

But i'm fine if you want to leave this as is in this PR :)